vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/vaadin-grid-flow-selection-column.html

```diff
@@ -3,12 +3,19 @@
 <dom-module id="vaadin-grid-flow-selection-column">
```

Comment:
v14 has duplicate implementations of the grid selection column for Polymer 3, and for Polymer 2 (Bower mode). Applied the same changes to the Polymer 2 version of the component here.

Apart from that the component, and the changes here, have diverged quite a bit from v23, but the changes are mostly simple.